packages/hooks/src/hooks/index.ts

```diff
@@ -0,0 +1 @@
+export * from './useWallet';
```

Comment:
One thing I've learned from experience is that it's better to be explicit with re-exports in `index` files:

```suggestion
export { useWallet } from './useWallet';
```

Anyone looking at this file can tell at a glance what is exported from this folder, whereas `export *` syntax requires them to read through each individual file to know what all is exported.